test/e2e/scenarios/no_np.go

```diff
@@ -0,0 +1,29 @@
+// Test the scenario where there are no network policies
+// implemented in which case all pods should be able to
+// communicate with each other
+
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] No network policy is implemented", func() {
+	PContext("When no network policy is defined", func() {
+		PIt("Should allow communication between all pods", func() {
+			By("creating listener pod 1 with label 1 in cluster 1 in namespace 1")
+			By("creating connecting pod 2 with label 2 in cluster 2 in namespace 2")
+			By("Testing connectivity between pod 1 and pod 2")
+		})
+	})
+
+	PContext("[Dataplane] When CNI does not support network policy", func() {
```

Comment:
use "network plugin" instead of CNI.   CNI is the caller interface between kubelet and the network plugin.